docker/build/installers/install_bazel.sh

```diff
@@ -24,7 +24,7 @@ cd "$(dirname "${BASH_SOURCE[0]}")"
 . installer_base.sh
 
 TARGET_ARCH=$(uname -m)
-VERSION="3.2.0"
+VERSION="3.3.0"
```

Comment:
Please also update the sha256sum values involved.
Note also for aarch64, we prebuilt bazel  binary of version 3.2.0, and hosted it on bcebos. So either keep version 3.2.0 for aarch64, or build one for 3.3.0 and host it on bcebos.

Reply:
done

Reply:
For aarch64, I'd like to keep current version and revisit when we have a machine :)